README.md

```diff
@@ -308,4 +308,4 @@ PyTriton follows the [Semantic Versioning](https://semver.org/) scheme for versi
 
 - [Changelog](CHANGELOG.md)
 - [Known Issues](https://triton-inference-server.github.io/pytriton/latest/known_issues)
-- [Contributing](CONTRIBUTING.md)
\ No newline at end of file
+- [Contributing](CONTRIBUTING.md)
```

Comment:
The pre-commit script should remove line-break at the end of file committed here.